.github/workflows/azureml-unit-tests.yml

```diff
@@ -19,6 +19,13 @@ on:
     branches:
       - 'staging'
       - 'main'
+    paths:
+      # Unit tests will be run only when there are changes in the
+      # unit tests related code including:
+      - examples/**
+      - recommenders/**
+      - tests/**
+      - setup.py
```

Comment:
Perhaps we can use [`paths-ignore`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) to avoid triggering unit tests when we only make changes to markdown files.

Reply:
Either allowing paths or ignoring files, we can use whatever works

Reply:
Looks like the folders mentioned above `examples`, `recommenders`, `tests`  have `.md` files in them. I believe any changes to those will trigger unit tests in the current state.

Reply:
Patched in the PR https://github.com/microsoft/recommenders/pull/1797